forge-gui/res/cardsfolder/c/composer_of_spring.txt

```diff
@@ -0,0 +1,9 @@
+Name:Composer of Spring
+ManaCost:1 G
+Types:Creature Satyr Bard
+PT:1/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Enchantment.YouCtrl | TriggerZones$ Battlefield | OptionalDecider$ You | Execute$ TrigPutIntoPlayEnch | TriggerDescription$ Constellation — Whenever an enchantment enters the battlefield under your control, you may put a land card from your hand onto the battlefield tapped. If you control six or more enchantments, instead you may put a creature or land card from your hand onto the battlefield tapped.
+SVar:TrigPutIntoPlayEnch:DB$ ChangeZone | ConditionCheckSVar$ X | ConditionSVarCompare$ LT6 | SubAbility$ TrigPutIntoPlayEnchCreature | Origin$ Hand | Destination$ Battlefield | Tapped$ True | ChangeType$ Land.YouCtrl
+SVar:TrigPutIntoPlayEnchCreature:DB$ ChangeZone | ConditionCheckSVar$ X | ConditionSVarCompare$ GE6 | Origin$ Hand | Destination$ Battlefield | Tapped$ True | ChangeType$ Creature.YouOwn,Land.YouOwn
```

Comment:
This should probably use Branch Effect

otherwise if the land is and enchantment, then it would increase the the Valid Enchantment count, making the second ability also true